### ISSUE TITLE
From 1.5.0 onwards, dashes don't collapse when extended

### ISSUE THREAD

**Issue by paleite** [simov/slugify#144]
1.5.0 broke the behavior when collapsing dashes in some cases:

I use the following test cases.

```
expect(slugify("day---night")).toBe("day-night"); // Works
expect(slugify("day   night")).toBe("day-night"); // Works
expect(slugify("day + night")).toBe("day-night"); // Broken in ^1.5.0, but working in <=1.4.7, when using extend as described below 
```

This is how I extended slugify.

```
slugify.extend({ "+": "-" }); // Broken in ^1.5.0, but working in <=1.4.7

// Note: The following works as expected, however: { "+": " " }
```

**Comment by Trott** [comment]
Looks like I broke this in #115.

FWIW, the `slug` module does what you want by default. No need to extend or anything like that. All three of your test strings above return `'day-night'` by default when passed to `slug()`.

**Comment by paleite** [comment]
Whoah, that was fast! 🚀 
Thanks 🙂

**Comment by Trott** [comment]
Fixed in slugify@1.6.4.
